groups.csv

```diff
@@ -1,2 +1,3 @@
 Group,Name,Institution,Department,Position,Email
 Groupless,Jed Brown,CU,CS,Faculty,jed.brown@colorado.edu
+Spring 2020:Anna and Valeria, Anna, CU, Earth Lab, student, pavlenkoanna2011@gmail.com
```

Comment:
Hey @annapav7 , can you please the group name?

Reply:
Ok!!!